render/board.go

```diff
@@ -2,11 +2,39 @@ package render
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/BattlesnakeOfficial/exporter/engine"
 	log "github.com/sirupsen/logrus"
 )
 
+type snakeCorner string
```

Comment:
types for this is nice because it improves devX for functions that take lots of strings. Now you can't mix up color, direction and corner